CppCoreGuidelines.md

```diff
@@ -17267,11 +17234,11 @@ Hard.
 
  `auto` is the weakest concept. Concept names convey more meaning than just `auto`.
 
-##### Example (using TS concepts)
+##### Example
 
-    vector<string> v{ "abc", "xyz" };
+    vector v{ .1, .2, .3 };
     auto& x = v.front();     // bad
-    String& s = v.front();   // good (String is a GSL concept)
+    floating_point auto& s = v.front();   // good (floating_point is a standard concept)
```

Comment:
Maybe label these examples as 'weak concept' and 'strong concept' instead of 'bad' and 'good'?  Depending on the rest of the context ```auto& x = v.front();``` may be 'good'.